package/src/components/CachedImages/CachedAvatar.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
cached avatar needs to be different because avatars dont belong to a message and urls are different